### ISSUE TITLE
Prevent detection of imports in comments (SCSS)

### ISSUE THREAD

**Issue by invliD** [andreyfedoseev/django-static-precompiler#81]
Bootstrap 4 currently contains an import in a comment (https://github.com/twbs/bootstrap/blob/1a6d0cc/scss/_reboot.scss#L47) that would create an infinite loop if it were outside a comment. Since the SCSS dependency detection ignores comments, it detects the import even though it shouldn't and thus creates the infinite loop: bootstrap.scss -> _reboot.scss -> bootstrap.scss -> …

The SCSS compiler needs to handle comments and ignore everything in there to prevent this bug in the future.

**Comment by andreyfedoseev** [comment]
@invliD Good catch, thank you!

**Comment by andreyfedoseev** [comment]
For anyone interested, at the moment, I don't have time to work on this issue myself. Pull requests with a fix are very welcome.

My proposed solution is to remove all comments from SCSS file before searching for `@import` statements. I added a stub implementation of `strip_comments` function to `SCSSCompiler`. I also made a set of (failing) tests for this function. The code is available in this branch: https://github.com/andreyfedoseev/django-static-precompiler/tree/fix-imports-in-comments
